1/incrementing-my-value.md

```diff
@@ -78,4 +78,4 @@ Remember to run `cargo +nightly test` to test your work.
 
 [embedded-code-previous](./assets/1.5-finished-code.rs ':include :type=code embed-previous')
 
-<!-- tabs:end -->
\ No newline at end of file
+<!-- tabs:end -->
```

Comment:
what changed here?

Reply:
@Robbepop really nothing! I was editting this file via github ui and it got changed.
Let me fix that.